### PR TITLE
Fix extract parse bug

### PR DESCRIPTION
**Pull request by yebrahim**
Also a minor bug with the environment variable getter.

Fixes https://github.com/googledatalab/datalab/issues/1486

### CODE REVIEW

File: google/datalab/bigquery/commands/_bigquery.py [hunk 1]
```diff
@@ -787,7 +787,8 @@ def _extract_cell(args, cell_body):
                          csv_delimiter=args['delimiter'], csv_header=args['header'],
                          compress=args['compress'])
   elif args['query'] or args['view']:
-    source = google.datalab.utils.commands.get_notebook_item(args['view'])
+    source_name = args['view'] if args['view'] else args['query']
```
Comment (ojarjur) [inline google/datalab/bigquery/commands/_bigquery.py]: This feels redundant to me.

What about using `source_name = args['view'] or args['query']` instead?

**Comment by parthea** [comment]
Is it possible to write a unit test that fails without the fix?

File: google/datalab/bigquery/commands/_bigquery.py (hunk 1 above)
Comment (yebrahim) [inline google/datalab/bigquery/commands/_bigquery.py]: done.

**Comment by yebrahim** [comment]
Fixed the unit tests to cover this case.

**Pull request merged**